Packages/com.vrchat.ClientSim/Runtime/CameraStacking/ClientSimStackedVRCameraSystem.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.XR;
+using VRC.Core;
+using VRC.SDK3.ClientSim;
+
+namespace VRC.SDK3.ClientSim
+{
+    public class ClientSimStackedVRCameraSystem : MonoBehaviour
+    {
+        public ClientSimStackedCamera[] CameraStack;
```

Comment:
Can we make this private?
```suggestion
        [SerializeField] private ClientSimStackedCamera[] CameraStack;
```

Reply:
yes, but I also need to update the asset afterwards